src/AzureRMTools.ts

```diff
@@ -137,7 +133,7 @@ export class AzureRMTools {
                 // If the documentUri is not in our dictionary of deployment templates, then we
                 // know that this document was opened (as opposed to changed/updated).
                 let stopwatch: Stopwatch;
-                if (!me._deploymentTemplates[documentUri]) {
+                if (!this._deploymentTemplates[documentUri]) {
```

Comment:
Don't these changes seem orthogonal to bumping the shared packages?

Reply:
I _have_ to change from `(this: IActionContext)` to `(actionContext: IActionContext)` because of a breaking change in the UI package (the build won't even pass if I don't do that). I suppose I could've left the `me` variable as-is, but the only reason we did that in the first place was because actionContext was `this`. Now that it's not `this`, it feels directly related to clean that up.